pkg/standalone-utils/test/VaultActions.test.ts

```diff
@@ -1196,4 +1206,678 @@ describe('VaultActions', function () {
       });
     });
   });
+
+  describe('stable pools', () => {
+    let poolIdStable: string;
```

Comment:
Given that this is so large, I'd consider moving it to its own file (`VaultActionsStableJoinExit.test.ts` or w/e)